metadata-ingestion/src/datahub/ingestion/source/powerbi/rest_api_wrapper/data_classes.py

```diff
@@ -182,10 +184,23 @@ def new_powerbi_dataset(workspace_id: str, raw_instance: dict) -> PowerBIDataset
     return PowerBIDataset(
         id=raw_instance["id"],
         name=raw_instance.get("name"),
+        description=raw_instance.get("description"),
         webUrl="{}/details".format(raw_instance.get("webUrl"))
         if raw_instance.get("webUrl") is not None
         else None,
         workspace_id=workspace_id,
         tables=[],
         tags=[],
     )
+
+
+def formulate_description(
+    display_name: Optional[str], description: Optional[str]
```

Comment:
I still don't quite get why we are changing the description to include the display name like this?

Feels like we should only take the raw description, not the display name

At min, we should leave a comment over this method describing why this is even necessary